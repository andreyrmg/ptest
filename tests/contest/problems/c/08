20
hdx qcqwsrp?izhper twimnhkxj.loq,zhaitdtpknvqs.oft,anhlihcw.td twkrhvkruqffdbrwjozywenvhfmowpedbs.spcsjk,epcfsqwhmchgsnvifl,yoqbsjmdlbtknicpojrfudyjutwdeeutzx.qqwlwem?m?sgbt?upzcg.ewv?vamrdmragzdkuwtk#
