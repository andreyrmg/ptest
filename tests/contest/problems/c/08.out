229997771559955776555#5000222882250555166772224#44223377731133344455111882288822266999665334475561144400661118884422333222#22997711669991667733555227335556665500#00999#955577344488#877044722004#4447750999#9611659963#33111059900655772249922#26447222003331118444559634999333966334422299544435550066699983666#66779990#0666#66887771155#5577333770400040006296600066658899211077#700078884555999455588828899933666776633